From 9170f28a0f6229fc5c4a9d147fab61db30ebecbf Mon Sep 17 00:00:00 2001
From: Dmitry Lavnikevich <dmitry.lavnikevich@edality.by>
Date: Mon, 6 May 2019 10:11:30 +0300
Subject: Add composite rcw-uboot image generation to mkfw flex-builder command

---
 tools/flex-builder | 28 +++++++++++++++++-----------
 1 file changed, 17 insertions(+), 11 deletions(-)

diff --git a/tools/flex-builder b/tools/flex-builder
index 121c89a..b907310 100755
--- a/tools/flex-builder
+++ b/tools/flex-builder
@@ -111,6 +111,7 @@ generate_trustbox_composite_firmware() {
 
 	local build_dir=$FBDIR/build/images/${MACHINE}
 	local fwimg=${build_dir}/${MACHINE}_${BOOTTYPE}_fw.itb
+	local bootloaderpblimg=${build_dir}/components/u-boot-with-pbl.bin
 	local rcwimg=`eval echo '${'"rcw_""${BOOTTYPE}"'}'`
 	local bootloaderimg=`eval echo '${uboot_'"${BOOTTYPE}"'boot}'`
 
@@ -167,6 +168,11 @@ generate_trustbox_composite_firmware() {
 	local bootloader_offset=`echo "$(( 0x00010000 / "$block_size" ))"`
 	local pfe_fw_offset=`echo "$(( 0x00240000 / "$block_size" ))"`
 	local ppa_fw_offset=`echo "$(( 0x00280000 / "$block_size" ))"`
+
+	dd if=/dev/zero bs=1M count=2 | tr '\000' '\377' > ${bootloaderpblimg}
+	dd if=${rcwimg} of=${bootloaderpblimg} obs=${block_size} seek=${rcw_offset} conv=notrunc
+	dd if=${bootloaderimg} of=${bootloaderpblimg} obs=${block_size} seek=${bootloader_offset} conv=notrunc
+
 	dd if=/dev/zero bs=1M count=64 | tr '\000' '\377' > ${fwimg}
 	dd if=${rcwimg} of=${fwimg} obs=${block_size} seek=${rcw_offset} conv=notrunc
 	dd if=${bootloaderimg} of=${fwimg} obs=${block_size} seek=${bootloader_offset} conv=notrunc
@@ -737,7 +743,7 @@ generate_distro_bootscr() {
 		    rm -f $FBDIR/$bootscript_enforce.tmp
 		fi
 		echo $securevalidate_fix > $FBDIR/$uboot_scr.tmp
-	    
+
        	    else
 		echo $securevalidate > $FBDIR/$uboot_scr.tmp
 	    fi
@@ -896,7 +902,7 @@ generate_bootpartition_tarball() {
 	imaevmopt="_imaevm"
 	if [ ! -f $FBDIR/packages/rfs/initrds/initramfs_imaevm_${DESTARCH}.img ]; then
 	    echo generating dependent $FBDIR/packages/rfs/initrds/initramfs_imaevm_${DESTARCH}.img ...
-	    flex-builder -i mkrfs -r buildroot:imaevm -a $DESTARCH -f $CONFIGLIST 
+	    flex-builder -i mkrfs -r buildroot:imaevm -a $DESTARCH -f $CONFIGLIST
 	fi
 	cp -f $FBDIR/packages/rfs/initrds/initramfs_imaevm_${DESTARCH}.img $bootpartdir/initramfs.img
 	for brd in $LS_MACHINE_LIST; do
@@ -967,13 +973,13 @@ generate_bootpartition_tarball() {
 	    if [ $brd = ls1012ardb ]; then
 		if [ ! -f $FBDIR/build/firmware/qoriq-engine-pfe-bin/ls1012a/u-boot/pfe_fw_sbl.itb ]; then
 		    flex-builder -c qoriq-engine-pfe-bin -f $CONFIGLIST
-		fi	
+		fi
 		cp -f build/firmware/qoriq-engine-pfe-bin/ls1012a/u-boot/pfe_fw_sbl.itb $bootpartdir/flash_images
 		fi
 		if [ $brd = trustbox ]; then
 		if [ ! -f $FBDIR/build/firmware/qoriq-engine-pfe-bin/trustbox/u-boot/pfe_fw_sbl.itb ]; then
 		    flex-builder -c qoriq-engine-pfe-bin -f $CONFIGLIST
-		fi	
+		fi
 		cp -f build/firmware/qoriq-engine-pfe-bin/trustbox/u-boot/pfe_fw_sbl.itb $bootpartdir/flash_images
 	    fi
 	    if [ $brd = ls1088ardb -o $brd = ls2088ardb -o $brd = lx2160ardb ]; then
@@ -1886,7 +1892,7 @@ flex_autobuild_all() {
 	build_rfs_apps imx
 	merge_apps_to_rfs imx
     fi
-    
+
     if [ "$BUILD_BUILDROOT_RFS" = "y" ]; then
 	flex-builder -i mkrfs -r buildroot:moderate -a $DESTARCH
 	flex-builder -i mklinux -r buildroot:moderate -a $DESTARCH
@@ -1920,7 +1926,7 @@ secure_sign_image() {
 	echo $FBDIR/packages/apps/cst not exist, build it ...
 	flex-builder -c cst -a $DESTARCH -f $CONFIGLIST
     fi
-    
+
     if [ ! -f $FBDIR/$uboot_scr ]; then
 	echo $FBDIR/$uboot_scr not exist, generating it ...
 	flex-builder -i mkdistroscr -m $1 -a $DESTARCH -f $CONFIGLIST
@@ -1963,7 +1969,7 @@ secure_sign_image() {
     fi
     cp $FBDIR/$device_tree $FBDIR/packages/apps/cst/uImage.dtb && echo "Copying dtb"
     cp $FBDIR/build/images/lsdk_linux_${DESTARCH}_${SOCFAMILY}_tiny.itb $FBDIR/packages/apps/cst/kernel.itb && echo "Copying kernel_itb"
-    
+
     if [ $1 = ls1043ardb -o $1 = ls1046ardb -o $1 = ls1012ardb -o $1 = ls1012afrwy -o $1 = trustbox ] && [ $DESTARCH = arm32 -a $SOCFAMILY = LS ]; then
 	kernelimg=$FBDIR/build/linux/kernel/arm32/LS/uImage.v8
     else
@@ -2020,15 +2026,15 @@ secure_sign_image() {
     if [ -f $FBDIR/$dpaa2_mc_fw ] ; then
 	cp $FBDIR/$dpaa2_mc_fw $FBDIR/packages/apps/cst/mc.itb
     fi
-    
+
     if [ -f $FBDIR/$dpaa2_mc_dpc ] ; then
 	cp $FBDIR/$dpaa2_mc_dpc $FBDIR/packages/apps/cst/dpc.dtb
     fi
-    
+
     if [ -f $FBDIR/$dpaa2_mc_dpl ] ; then
 	cp $FBDIR/$dpaa2_mc_dpl $FBDIR/packages/apps/cst/dpl.dtb
     fi
-    
+
     if [ ! -d  $FBDIR/build/firmware/secboot_hdrs/$1 ] ; then
 	mkdir -p  $FBDIR/build/firmware/secboot_hdrs/$1
     fi
@@ -2053,7 +2059,7 @@ secure_sign_image() {
     cp $FBDIR/packages/apps/cst/hdr_linux.out $FBDIR/build/firmware/secboot_hdrs/$1
     if [  $1 = ls1012afrwy ] ; then
 	cp $FBDIR/packages/apps/cst/hdr_kernel.out $FBDIR/build/firmware/secboot_hdrs/$1
-    fi    
+    fi
     cp $FBDIR/packages/apps/cst/hdr_bs.out $FBDIR/build/firmware/secboot_hdrs/$1/hdr_${1}_bs.out
     cp $FBDIR/packages/apps/cst/srk_hash.txt $FBDIR/build/images
     cp $FBDIR/packages/apps/cst/srk.pri $FBDIR/build/images
-- 
2.20.1

