From 63792ff5778cc54dbe7a4fba27efd048cbb16b92 Mon Sep 17 00:00:00 2001
From: Dmitry Lavnikevich <dmitry.lavnikevich@scalys.by>
Date: Tue, 28 Jul 2020 17:20:45 +0300
Subject: [PATCH] trustbox: Update Linux source tree head

---
 configs/trustbox.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/trustbox.cfg b/configs/trustbox.cfg
index 202dd92..2bd8dfb 100644
--- a/configs/trustbox.cfg
+++ b/configs/trustbox.cfg
@@ -199,7 +199,7 @@ linux_firmware_repo_branch=master
 #
 linux_repo_url=https://github.com/Scalys/linux-qoriq.git
 linux_repo_branch=trustbox-2004
-linux_repo_commit=1d6cc0257f9e0e2eedff67b1b9e739f0a7efd0d6
+linux_repo_commit=a12f516a789ae4bd5404e48267eaa96764d265ba
 second_linux_repo_tag=LSDK-20.04-V4.19
 cryptodev_linux_repo_url=https://source.codeaurora.org/external/qoriq/qoriq-components/cryptodev-linux.git
 cryptodev_linux_repo_tag=LSDK-20.04
-- 
2.27.0

