From a2c53f3dd453ae33944d9ab415bc42a40e7244cb Mon Sep 17 00:00:00 2001
From: Dmitry Lavnikevich <dmitry.lavnikevich@edality.by>
Date: Wed, 15 May 2019 17:06:25 +0300
Subject: Enable temporary fix for docker operation

---
 configs/trustbox.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/trustbox.cfg b/configs/trustbox.cfg
index dcd077a..6907524 100644
--- a/configs/trustbox.cfg
+++ b/configs/trustbox.cfg
@@ -115,7 +115,7 @@ linux_config_list_arm32v7="multi_v7_defconfig multi_v7_lpae.config lsdk.config"
 #kernel git repository
 linux_repo_url=https://github.com/Scalys/linux-qoriq.git
 linux_repo_branch=trustbox-1903
-linux_repo_commit=a43402647a425982b76fb9a559e11b5e78e28dfe
+linux_repo_commit=139433451c698f7726228ef7b5098d25cbd5e425
 
 # uboot git repository
 u_boot_repo_url=https://github.com/Scalys/u-boot-qoriq.git
-- 
2.21.0

