From b9154f350772a20d745f000bf67c25923b4b59f1 Mon Sep 17 00:00:00 2001
From: Nicholas Pridybailo <nikolai.pridybailo@edality.by>
Date: Thu, 16 May 2019 10:04:13 +0300
Subject: Back to stable u-boot commit

---
 configs/trustbox.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configs/trustbox.cfg b/configs/trustbox.cfg
index 6907524..3dff84a 100644
--- a/configs/trustbox.cfg
+++ b/configs/trustbox.cfg
@@ -120,7 +120,7 @@ linux_repo_commit=139433451c698f7726228ef7b5098d25cbd5e425
 # uboot git repository
 u_boot_repo_url=https://github.com/Scalys/u-boot-qoriq.git
 u_boot_repo_branch=trustbox-1903
-u_boot_repo_commit=6796103d3f20bdaac19e1d36d2b222d08a88ea2c
+u_boot_repo_commit=655504ca1a690c18fa68615658d62d74438dee2c
 
 #linux modules git repository
 lttng_modules_repo_url=git://git.lttng.org/lttng-modules.git
-- 
2.21.0

