From ad22d2372ede8a870f11916d77e8c20ea04eab06 Mon Sep 17 00:00:00 2001
From: Nicholas Pridybailo <nikolai.pridybailo@edality.by>
Date: Mon, 3 Jun 2019 13:05:54 +0300
Subject: Compiling ppa firmware with enabled optee

---
 packages/firmware/Makefile | 16 ++++++++++++++--
 1 file changed, 14 insertions(+), 2 deletions(-)

diff --git a/packages/firmware/Makefile b/packages/firmware/Makefile
index 94a689b..f36e94d 100644
--- a/packages/firmware/Makefile
+++ b/packages/firmware/Makefile
@@ -166,10 +166,22 @@ ppa-generic:
 ifeq ($(MACHINE), trustbox)
 ifeq ($(CONFIG_BUILD_PPA), y)
 	@$(call fbprint_b,"ppa-generic")
+        ifeq ($(CONFIG_BUILD_OPTEE), y)
+		@if [ ! -f $(FBDIR)/build/firmware/optee/tee_$(MACHINE).bin -o ! -d $(FBDIR)/packages/apps/optee_os ]; then \
+			$(call fbprint_n,"Optee-OS required and not built yet. Building...");\
+			cd $(FBDIR) && flex-builder -c optee_os -a $(DESTARCH) -m $(MACHINE);\
+		fi;
+        endif
 	@$(call fetch-git-tree,$(ppa_generic_repo))
 	@test -d $(FBDIR)/build/firmware/ppa-generic || mkdir -p $(FBDIR)/build/firmware/ppa-generic/$(MACHINE)
-	@cd $(FBDIR)/packages/firmware/ppa-generic/ppa/ && ./build prod rdb-fit ls1012 && \
-	cp soc-ls1012/build/obj/ppa.itb $(FBDIR)/build/firmware/ppa-generic/$(MACHINE);
+	@cd $(FBDIR)/packages/firmware/ppa-generic/ppa/ && ./build clean ls1012
+        ifeq ($(CONFIG_BUILD_OPTEE), y)
+		@cp $(FBDIR)/build/firmware/optee/tee_$(MACHINE).bin $(FBDIR)/packages/firmware/ppa-generic/ppa/soc-ls1012/tee.bin
+		@cd $(FBDIR)/packages/firmware/ppa-generic/ppa/ && ./build prod rdb-fit spd=on ls1012
+        else
+		@cd $(FBDIR)/packages/firmware/ppa-generic/ppa/ && ./build prod rdb-fit ls1012
+        endif
+	@cp $(FBDIR)/packages/firmware/ppa-generic/ppa/soc-ls1012/build/obj/ppa.itb $(FBDIR)/build/firmware/ppa-generic/$(MACHINE)
 	@$(call fbprint_d,"ppa-generic")
 endif
 endif
-- 
2.21.0

