From 064b10c812c31254ed1f64becd9d9529aa098a3e Mon Sep 17 00:00:00 2001
From: Nicholas Pridybailo <nikolai.pridybailo@edality.by>
Date: Mon, 10 Jun 2019 17:03:58 +0300
Subject: Set trustbox hostname

---
 tools/flex-builder | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tools/flex-builder b/tools/flex-builder
index 6e6d735..41cbc27 100755
--- a/tools/flex-builder
+++ b/tools/flex-builder
@@ -1043,6 +1043,7 @@ build_distro_rfs_ubuntu() {
     elif [ $DESTARCH = ppc64 ]; then
 	tarch=ppc64el
     fi
+
     if [ $DISTROTYPE = ubuntu -o $DISTROTYPE = debian ] && [ -f $RFSDIR/etc/.buildinfo ]; then
 	echo $RFSDIR already exists!
 	exit 0
@@ -1092,6 +1093,10 @@ build_distro_rfs_ubuntu() {
 	flex-mkdistrorfs $tarch $CODENAME $DISTROSCALE $pkglist $KEEP_LEGACY_PKG
     fi
 
+    if [ "$MACHINE" = "trustbox" ]; then
+        echo $MACHINE > $RFSDIR/etc/hostname
+    fi
+
     if [ -d $RFSDIR/etc/udev/rules.d ]; then
 	sudo cp -f $FBDIR/packages/rfs/misc/udev/udev-rules-qoriq/72-fsl-dpaa-persistent-networking.rules $RFSDIR/etc/udev/rules.d
     fi
-- 
2.21.0

