From 74f4c0c7a38718e57eb4f2c3982437a2556f5410 Mon Sep 17 00:00:00 2001
From: Dmitry Lavnikevich <dmitry.lavnikevich@edality.by>
Date: Mon, 10 Jun 2019 17:48:16 +0300
Subject: Update iotedge packages version

---
 configs/trustbox.cfg | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configs/trustbox.cfg b/configs/trustbox.cfg
index 7e2684b..ebd3db1 100644
--- a/configs/trustbox.cfg
+++ b/configs/trustbox.cfg
@@ -200,10 +200,10 @@ buildroot_repo_branch=2018.11.x
 ramdiskrfs_url=https://www.nxp.com/lgfiles/sdk/lsdk
 ruby_src_url=https://cache.ruby-lang.org/pub/ruby/ruby-2.5.1.tar.gz
 iproute2_src_url=https://mirrors.edge.kernel.org/pub/linux/utils/net/iproute2/iproute2-4.15.0.tar.gz
-azure_iotedge_deb_name=iotedge_1.0.8~dev-1_arm64.deb
-azure_iotedge_deb_url=https://www.grapeboard.com/wp-content/pkgs/lsdk-1903/iotedge_1.0.8~dev-1_arm64.deb
-azure_libiothsm_deb_name=libiothsm-std_1.0.8~dev-1_arm64.deb
-azure_libiothsm_deb_url=https://www.grapeboard.com/wp-content/pkgs/lsdk-1903/libiothsm-std_1.0.8~dev-1_arm64.deb
+azure_iotedge_deb_name=iotedge_1.0.8~rc1-1_arm64.deb
+azure_iotedge_deb_url=https://www.grapeboard.com/wp-content/pkgs/lsdk-1903/iotedge_1.0.8~rc1-1_arm64.deb
+azure_libiothsm_deb_name=libiothsm-std_1.0.8~rc1-1_arm64.deb
+azure_libiothsm_deb_url=https://www.grapeboard.com/wp-content/pkgs/lsdk-1903/libiothsm-std_1.0.8~rc1-1_arm64.deb
 ftpm_repo_url=https://github.com/Microsoft/ms-tpm-20-ref.git
 ftpm_repo_commit=d979cae6db01ea2e57dbce92d700ce1d0edf78fa
 optee_examples_repo_url=https://github.com/linaro-swg/optee_examples.git
-- 
2.21.0

